packages/wrangler/package.json

```diff
@@ -47,8 +47,8 @@
   "devDependencies": {
     "@babel/types": "^7.16.0",
     "@iarna/toml": "^2.2.5",
-    "@types/mime": "^2.0.3",
     "@types/estree": "^0.0.50",
+    "@types/mime": "^2.0.3",
```

Comment:
This change is because `npm i` reordered the dependencies into alphabetical order.

Reply:
Which is good and can be annoyingly funny.